conf/script/build-ha-csm

```diff
@@ -190,8 +190,6 @@ csm_rsc_add() {
 
     sudo pcs -f $cib_file constraint colocation add csm-kibana with els-search-clone \
```

Comment:
This colocation constraint between csm and elastic search also needs to be removed.

Reply:
Good finding. Thanks.